src/test/java/uk/nickbdyer/tictactoe/GameTest.java

```diff
@@ -40,7 +43,8 @@ public void gameKnowsWhosTurnItIs() {
     }
 
     private void makeMultipleMoves(int numberOfMoves, int[] positions) {
-        game = new Game(new PlayerFactory(new MockUI()).create(GameType.HvsH));
+        PlayerFactory pf = new PlayerFactory(new MockUI());
+        game = new Game(Arrays.asList(pf.create(H, X), pf.create(H, O)));
```

Comment:
`pf.create(H, X), pf.create(H, O)` is a bit confusing because of all those one-letter enums.
Changing `H` to `Human` might improve readability.